libs/auth/src/jurisdictions.ts

```diff
@@ -2,7 +2,7 @@
  * The jurisdiction in all dev certs. Follows the format of a prod cert jurisdiction
  * ({state-2-letter-abbreviation}.{county-or-town}).
  */
-export const DEV_JURISDICTION = 'st.dev-jurisdiction';
+export const DEV_JURISDICTION = 'vx.test';
```

Comment:
As noted in the commit message, I'm switching the dev jurisdiction to match what we use for prod QA images (which in a separate PR I'm going to make more similar to dev, in terms of keys/certs used).

I'm adding this to this PR because I'm already having to regenerate dev keys/certs for the machine ID change. And I wanted to lump in this jurisdiction change to avoid yet another change to dev keys/certs after this PR.

Though I think that few engineers are actually still using physical smart cards in dev (as opposed to the dev dock), I'll still be sure to note that dev smart cards will need to be reprogrammed after this PR.